sdk/core/core-tracing/lib/wrappers/opencensus/openCensusTraceWrapper.ts

```diff
@@ -3,27 +3,24 @@
 import { Tracer } from "../../interfaces/tracer";
 import { SpanOptions } from "../../interfaces/SpanOptions";
 import { Span } from "../../interfaces/span";
-import { OpenCensusSpanPlugin } from "../opencensus/openCensusSpanPlugin";
-import { SupportedPlugins } from '../../utils/supportedPlugins';
+import { OpenCensusSpanWrapper } from "./openCensusSpanWrapper";
 import { BinaryFormat } from "../../interfaces/BinaryFormat";
 import { HttpTextFormat } from "../../interfaces/HttpTextFormat";
+import { Tracer as OpenCensusTracer } from "../../interfaces/OpenCensus/model";
```

Comment:
Did you consider pulling in `@opencensus/core` as a dependency instead of copying the types? Do you foresee any issues with customers using a different version of `@opencensus` than what our interfaces represent, and getting type errors?

I'd also want to avoid depending on it since that would pull in other dependencies like continuation-local-storage as well, though if we're only using the types we shouldn't import any of its dependencies at runtime. If there's no concern around versioning then ignore this comment.

Reply:
cc @bterlson for a recap on earlier discussions on what dependencies we want to have and what not in core-tracing

Reply:
I found a package from opencensus that has just their types. This seems like a fine/easy dependency to take.